SeaPublicWebsite/Views/EnergyEfficiency/GlazingType.cshtml

```diff
@@ -39,27 +39,30 @@
                         </p>
                      </text>;
                 
-                Func<object, object> epcHint = null;
+                Func<object, object> blueBannerHint = null;
                 if (Model.Epc?.GlazingType is not null)
                 {
-                    Func<object, object> suggestion = Model.Epc?.GlazingType switch {
-                        GlazingType.SingleGlazed => 
-                            @<text>
-                                all your windows are <strong>single glazed</strong>.
-                             </text>,
-                        GlazingType.Both => 
-                            @<text>
-                                at least one of your windows is <strong>single glazed</strong>.
-                             </text>,
-                        GlazingType.DoubleOrTripleGlazed => 
-                            @<text>
-                                all your windows are <strong>double, triple or secondary glazed</strong>.
-                             </text>,
-                        _ => throw new ArgumentOutOfRangeException()};
-                    epcHint = 
+                    blueBannerHint = 
                         @<p class="govuk-body">
                             Your @Model.Epc.LodgementYear Energy Performance Certificate suggests 
-                            @await Html.GovUkHtmlText(new HtmlText(suggestion, null))
+                            <strong>
+                                @(Model.Epc?.GlazingType switch {
+                                    GlazingType.SingleGlazed => "all your windows are single glazed.",
+                                    GlazingType.Both => "at least one of your windows is single glazed.",
+                                    GlazingType.DoubleOrTripleGlazed => "all your windows are double, triple or secondary glazed.",
+                                    _ => throw new ArgumentOutOfRangeException()})
+                            </strong>
+                        </p>;
+                } else if (Model.YearBuilt is not null and not YearBuilt.DoNotKnow)
+                {
+                    blueBannerHint =
+                        @<p class="govuk-body">
+                            Windows in homes of your type
+                            <strong>
+                                @(Model.YearBuilt < YearBuilt.From1983To1995 
```

Comment:
This sort of logic really doesn't belong in a cshtml file. It's business logic, so it should really be in the PropertyData class (where it can have unit tests too).
e.g. something like `bool PropertyData.HintHasSingleGlazing` which can then go into the view model and be checked here.

Reply:
Yes, sorry if that wasn't clear. This comment applies to all the similar logic in the cshtml files